src/tokens.scss

```diff
@@ -0,0 +1,5 @@
+:host {
+  --ds-auro-drawer-background-color: var(--ds-color-background-primary-100-default, $ds-color-background-primary-100-default);
+  --ds-auro-drawer-text-color: var(--ds-color-text-primary-default, $ds-color-text-primary-default);
```

Comment:
Shouldn't this just start as `inherit`?

Reply:
Don't think so. It was not `inherit` before and the blueprint shows using these tokens